src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-cluster-details/nfs-cluster-details.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'cd-nfs-cluster-details',
+  templateUrl: './nfs-cluster-details.component.html',
+  styleUrls: ['./nfs-cluster-details.component.scss']
+})
+export class NfsClusterDetailsComponent {
+  title = $localize`Export`;
+  @Input()
+  selection: any;
```

Comment:
should the type be `CdTableSelection`?
```suggestion
  selection: CdTableSelection;
```

Reply:
Updated